docs/install_linux.md

```diff
@@ -3,61 +3,89 @@ Welcome to the installation guide for WildBeast on Linux! In this guide, we'll w
 ##Prerequisites
 - A Linux server
 	-  Type: VPS is enough to run WB, dedicated is a bit overshooting unless you have something else running there as well.
-	-  OS: Ubuntu, latest version if possible.
-	-  RAM: 512 MB upwards, if you plan on having it on just a few servers. Scale up if necessary.
-	-  Processor: Single core @ 3,30 GHz will work just fine, as above.
+	-  OS: Ubuntu LTS or latest, or Debian 8
```

Comment:
Using `or` to initiate a subordinate clause is a bit clumsy, I'd say having `optionally` is better from a grammar standpoint.